public/themes/default/stylesheets/style.css

```diff
@@ -32,6 +32,14 @@ a {
 #locale, #setting_dns select {
   border: 1px solid #cccccc; }
 
+.hover-style{
+	cursor: pointer;
+}
+
+.hover-style:hover{
+	background: ##d8d8d873;	
```

Comment:
Is  ##d8d8d873 a valid color?

Reply:
No it is not. Thanks for pointing it out. i pushed the changes.